### PR TITLE
removed unused shelf from template

### PR DESCRIPTION
**Pull request by denisebiscaro**
#### What problem is this solving?

Remove unused shelf block from home template. It might confuse users.

See workspace:
https://denise--storecomponents.myvtex.com/

### CODE REVIEW

File: store/blocks/home/home.jsonc [hunk 1]
```diff
@@ -15,10 +15,6 @@
     ]
   },
 
-  "shelf#home": {
```
Comment (guerreirobeatriz) [inline store/blocks/home/home.jsonc:18]: Don't we need to declare the `list-context.product-list` instead? 🤔

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
```
Comment (klzns) [inline CHANGELOG.md:9]:
```suggestion
### Fixed
```

**Comment by denisebiscaro** [comment]
> Missing linked workspace

https://denise--storecomponents.myvtex.com/

File: store/blocks/home/home.jsonc (hunk 1 above)
Comment (denisebiscaro) [reply store/blocks/home/home.jsonc:18]: noo.. this block is only declared but it is not being used anywhere. The `list-context.product-list` is already declared properly

**Comment by klzns** [comment]
Fixed on #231

**Pull request closed without merging**